scripts/sommelier

```diff
@@ -183,9 +183,19 @@ start_app() {
     . "$SNAP/sommelier/hooks/pre-start"
   fi
 
+  if [[ -e "${ACTION_ARGS}" ]]; then
+    # Convert Unix paths to Windows paths.
+    for i in "${ACTION_ARGS[@]}"; do
+      args+=( "$(winepath -w "$i")" )
+    done
+    APP_ARGS=( "${args[@]}" )
```

Comment:
This can simply be `ACTION_ARGS=( "${args[@]}" )`, so you don't have to change lines 198,228 and 231. Then we also don't risk on accidentally using the wrong array somewhere down the line.